io.gitlab.o20.word.json

```diff
@@ -0,0 +1,26 @@
+{
+    "id": "io.gitlab.o20.word",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.13",
+    "sdk": "org.kde.Sdk",
+    "command": "word64.run",
+    "finish-args": ["--share=ipc", "--socket=x11", "--socket=wayland", "--device=dri", "--filesystem=xdg-documents/O20"],
```

Comment:
> "--filesystem=xdg-documents/O20"

You'll want `--filesystem=xdg-documents/O20:create` as the directory generally won't exist.